detection-rules/attachment_callback_scam.yml

```diff
@@ -0,0 +1,31 @@
+name: "Callback scam with attachment (first-time sender)"
+description: |
+    An invoice or receipt attachment from a freemail sender which is classified by the NLU engine as a Callback Scam.
```

Comment:
this rule currently isn't filtered to freemail senders, should it be?

Reply:
I think we can leave it out